src/wp-admin/includes/ajax-actions.php

```diff
@@ -3975,18 +3975,37 @@ function wp_ajax_crop_image() {
 
 			$size       = wp_getimagesize( $cropped );
 			$image_type = ( $size ) ? $size['mime'] : 'image/jpeg';
+			/** @var WP_Post $original_attachment */
+			$original_attachment = get_post( $attachment_id );
+			$has_title           = '' !== trim( $original_attachment->post_title );
 
 			$object = array(
-				'post_title'     => wp_basename( $cropped ),
+				// Copy the image title attribute (post_title field) from the original image.
+				'post_title'     => $has_title ? $original_attachment->post_title : wp_basename( $cropped ),
 				'post_content'   => $url,
 				'post_mime_type' => $image_type,
 				'guid'           => $url,
 				'context'        => $context,
 			);
 
+			// Copy the image description attribute (post_content field) from the original image.
+			if ( '' !== trim( $original_attachment->post_content ) ) {
+				$object['post_content'] = $original_attachment->post_content;
+			}
```

Comment:
Instead of assigning the `post_content` above and then overwriting it, move this inline into the array above.

Reply:
Currently updating and will push shortly